src/interaction/InteractionManager.js

```diff
@@ -701,7 +705,13 @@ InteractionManager.prototype.processTouchEnd = function ( displayObject, hit )
         if( displayObject._touchDown )
         {
             displayObject._touchDown = false;
-            this.dispatchEvent( displayObject, 'tap', this.eventData );
+            
+            // calculate the distance since the tap
+            var tapDistance = displayObject._touchDownPoint.distance(this.eventData.data.global);
```

Comment:
As for @englercj comment in the next file, I think we should keep consistency and make this a tapRadius.